src/templates/admin/repository/submit/files.html

```diff
@@ -27,12 +27,17 @@ <h2>Information</h2>
                 <div class="content">
                     <p>
                         {% trans 'You can use the form below to select and upload your manuscript file.' %}
+                        <a href="#" data-open="submission_information" aria-controls="submission_information" aria-haspopup="true" tabindex="0">Review submission requirements.</a>
                     </p>
                     {% if request.repository.limit_upload_to_pdf %}
-                        <p>{{ request.repository.name }} {% trans 'requires that all author uploads be PDF files.' %}</p>
+                        <p>{{ request.repository.name }} {{ request.repository.require_pdf_help }}</p>
                     {% else %}
                         <p>{% trans 'You can upload a file in any format though PDF is recommended.' %}</p>
```

Comment:
The new flexibility for other parts of this page is great. I wish this line were removed though, since it could potentially conflict with the custom text when someone doesn't require PDFs but wants to give other instructions. Also I don't think users need to be told any file type is allowed, since the upload will be a smooth process in that case. What if we remove the line altogether?

Reply:
> The new flexibility for other parts of this page is great. I wish this line were removed though, since it could potentially conflict with the custom text when someone doesn't require PDFs but wants to give other instructions. Also I don't think users need to be told any file type is allowed, since the upload will be a smooth process in that case. What if we remove the line altogether?

Agreed. Commit incoming.